packages/amplify-e2e-core/src/categories/auth.ts

```diff
@@ -916,3 +916,81 @@ export function addAuthWithMaxOptions(cwd: string, settings: any) {
       });
   });
 }
+
+//add default auth
+export function addAuth(projectDir: string) {
+  return new Promise((resolve, reject) => {
+    spawn(getCLIPath(), ['add', 'auth'], { cwd: projectDir, stripColors: true })
+      .wait('Do you want to use the default authentication and security configuration')
+      .sendCarriageReturn()
+      .wait('How do you want users to be able to sign in')
+      .sendCarriageReturn()
+      .wait('Do you want to configure advanced settings')
+      .sendCarriageReturn()
+      .wait('"amplify publish" will build all your local backend and frontend resources')
+      .run((err: Error) => {
+        if (!err) {
+          resolve();
+        } else {
+          reject(err);
+        }
+      });
+  });
+}
+
+//update auth to add user group
+export function updateAuthAddFirstUserGroup(projectDir: string, groupName: string) {
```

Comment:
Can this function accept an list of strings for group name
```suggestion
export function updateAuthWithUserpoolGroups(projectDir: string, groupNames: string[]) {
```